keycloak/docker-compose.yml

```diff
@@ -5,7 +5,7 @@ services:
     healthcheck:
       test: ['CMD', 'curl', '-f', 'http://localhost:8080/auth']
       start_period: 60s
-    image: jboss/keycloak:15.0.2
```

Comment:
The version was originally set to match production. Can we ensure it is still the same as production

Reply:
Ah, it is almost certainly not 😐  I’ll revert this